test/unit/cookiejar.test.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+process.env.NODE_DEBUG = 'axios';
+jest.mock('axios');
+const axios = require('axios');
+const mockAxiosInstance = jest.fn();
+const tough = require('tough-cookie');
+mockAxiosInstance.defaults = {};
+mockAxiosInstance.interceptors = {
+  request: {
+    use: jest.fn(),
+  },
+  response: {
+    use: jest.fn(),
+  },
+};
+axios.default.create.mockReturnValue(mockAxiosInstance);
+
+const { RequestWrapper } = require('../../dist/lib/request-wrapper');
+const requestWrapperInstance = new RequestWrapper({ jar: new tough.CookieJar() });
+
+describe('axios', () => {
+  let env;
+  beforeEach(function() {
+    jest.resetModules();
+    env = process.env;
+    process.env = {};
+  });
+  afterEach(function() {
+    process.env = env;
+  });
+  it('should enable debug', () => {
+    // these should have been called when requestWrapperInstance and when the cookiejar wrapper was instantiated
+    expect(mockAxiosInstance.interceptors.request.use).toHaveBeenCalledTimes(2);
+    expect(mockAxiosInstance.interceptors.response.use).toHaveBeenCalledTimes(2);
```

Comment:
What is the idea behind these tests? It seems like the interceptors and the cookie support should be unrelated

Reply:
One thing that happens when we wrap the instance the second time is that the interceptors are called twice - one on the initial wrapping and further on the  cookie wrapping.  This is not essential, but it is a good test that the instance is wrapped a second time.

Reply:
Okay, I think I see where you're coming from. Will you update the comment you left to explain that a little more? I was confused on first read